tests/src/runner/podman-desktop-runner.ts

```diff
@@ -77,6 +79,47 @@ export class PodmanDesktopRunner {
     }
   }
 
+  async getApplicationState(): Promise<{
+    isVisible: boolean;
+    isDevToolsOpened: boolean;
+    isCrashed: boolean;
+    isMaximized: boolean;
+  }> {
+    const window: JSHandle<BrowserWindow> = await this.getBrowserWindow();
+
+    return await window.evaluate(
+      (
+        mainWindow,
+      ): Promise<{ isVisible: boolean; isDevToolsOpened: boolean; isCrashed: boolean; isMaximized: boolean }> => {
+        const getState = () => ({
+          isVisible: mainWindow.isVisible(),
+          isDevToolsOpened: mainWindow.webContents.isDevToolsOpened(),
+          isCrashed: mainWindow.webContents.isCrashed(),
+          isMaximized: mainWindow.isMaximized(),
+        });
+
+        return new Promise(resolve => {
+          resolve(getState());
+        });
+      },
+    );
+  }
+
+  async setupApplication(): Promise<void> {
+    const window: JSHandle<BrowserWindow> = await this.getBrowserWindow();
+
+    await window.evaluate((mainWindow): Promise<void> => {
+      mainWindow.webContents.closeDevTools();
+      mainWindow.maximize();
+      mainWindow.webContents.openDevTools({ mode: 'detach' });
```

Comment:
do we need to close and reopen or opening in detached mode works ?

maybe we should do it there https://github.com/containers/podman-desktop/blob/eaa8a87afdff845ea1c14a24cf7682afb0094d6b/packages/main/src/mainWindow.ts#L97-L99 (I'm using detach mode by default) or checking if CI is enabled and open it there

Reply:
I recall trying that, but it seemed to be a problem on ci. I will double check.

Reply:
It works without closing. but for now only locally. On GHA it does something wrong. I would wait for other PRs to get merged so I can see what is going on.